pkg/bpf/endpoint.go

```diff
@@ -67,7 +66,7 @@ func (k EndpointKey) ToIP() net.IP {
 // String provides a string representation of the EndpointKey.
 func (k EndpointKey) String() string {
 	if ip := k.ToIP(); ip != nil {
-		return net.JoinHostPort(ip.String(), fmt.Sprintf("%d", k.Key))
+		return net.JoinHostPort(ip.String(), fmt.Sprintf("%d", k.Family))
```

Comment:
Given that the `ip` should be a `net.IP` which clearly formats IPv4 differently from IPv6, I think it is sufficient to simply print the IP address and avoid printing the key or family here.

```suggestion
		return ip.String
```

Reply:
done!